Exercises.String/Exercises.String/GoogleMapsModel.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Exercises.String
+{
+    class GoogleMapsModel
+    {
+        public Dictionary<string, List<string>> UrlStruct { get; }=new Dictionary<string, List<string>>();
```

Comment:
Please surround `=` with whitespaces. Same for `+`, non unary `-`, `/`, `*`, etc. `ctrl+k+d` should do it for you